shared/engine/index.js

```diff
@@ -283,7 +283,7 @@ class Engine {
     } else if (method === 'keybase.1.logUi.log' && this._hasNoHandler(method, callMap || {}, this._generalIncomingRpc)) {
       log(param)
       wrappedResponse.result()
-    } else if (!sessionID && this.generalListeners[method]) {
```

Comment:
Sorry should have mentioned this in the PR. This RPC sets a sessionID to be non-zero. I meant to mention this in #core but forgot. Will do.